### PR TITLE
Ignition EA

### PR DESCRIPTION
**Pull request by mxiao-cll**
## Description

Add EA to PoR as well

## Changes

- High level
- changes that
- you made

## Steps to Test

1. Steps
2. to
3. test

## Quality Assurance

- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `adapter-secrets` configuration file or update the [soak testing blacklist](/packages/scripts/src/get-changed-adapters/soakTestBlacklist.ts).
- [ ] If a new adapter was made, or a new endpoint was added, update the `test-payload.json` file with relevant requests.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Jira.
- [ ] This is related to a maximum of one Jira story or GitHub issue.
- [ ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types).
- [ ] All code changes have 100% unit and integration test coverage. If testing is not applicable or too difficult to justify doing, the reasoning should be documented explicitly in the PR.

### CODE REVIEW

File: packages/sources/ignition-address-list/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@chainlink/ignition-address-list-adapter",
+  "version": "0.0.0",
```
Comment (mmcallister-cll) [inline packages/sources/ignition-address-list/package.json:3]: nit: should we make this v1?

File: packages/sources/ignition-address-list/src/config/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  API_KEY: {
+    description: 'An API key for Data Provider',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_ENDPOINT: {
+    description: 'An API endpoint for Data Provider',
+    type: 'string',
+    sensitive: true,
```
Comment (mmcallister-cll) [inline packages/sources/ignition-address-list/src/config/index.ts:13]: nit: is this really sensitive?

File: packages/sources/ignition-address-list/package.json (hunk 1 above)
Comment (mxiao-cll) [reply packages/sources/ignition-address-list/package.json:3]: changeset will automatically bump it to v1

File: packages/sources/ignition-address-list/src/config/index.ts (hunk 2 above)
Comment (mxiao-cll) [reply packages/sources/ignition-address-list/src/config/index.ts:13]: I think sensitive means case sensitive?

File: packages/sources/ignition-address-list/src/transport/address.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          },
+        }
+      }
+      const addresses = response.data.result
+        .map((r) => ({
+          address: r.address,
+          network: param.network,
+          chainId: param.chainId,
+        }))
+        .sort((a, b) => a.address.localeCompare(b.address))
```
Comment (mmcallister-cll) [inline packages/sources/ignition-address-list/src/transport/address.ts:50]: lol nice

**Review by mmcallister-cll — APPROVED** [review]
LGTM, just sanity check that por-indexer is OK to accept the different length addresses that are returned in this list please 🙏

File: packages/sources/ignition-address-list/src/config/index.ts (hunk 2 above)
Comment (mxiao-cll) [reply packages/sources/ignition-address-list/src/config/index.ts:13]: I am wrong

->

// Set to true to censor this env var from logs. Default = false

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -82,6 +82,7 @@ jobs:
       id-token: write
       contents: read
     strategy:
+      max-parallel: 5
```
Comment (mxiao-cll) [inline .github/workflows/deploy.yml:85]: Looks like we are hitting docker push limit so slow down the push a bit

**Review by mmcallister-cll — APPROVED** [review]

**Pull request merged**